eslint-rules/tests/lib/rules/component-prop-deprecation.js

```diff
@@ -89,6 +89,20 @@ ruleTester.run('component-prop-deprecation', rule, {
       <Picker value="value" migrate={true}/>
       `
     },
+    {
+      options: ruleOptions,
+      code: 'import {Picker} from \'module-with-deprecations\'; <Picker t="title" s="subtitle" migrate/>',
+      errors: [
```

Comment:
If that's a valid example why the error? (Same for the next entry)

Reply:
10x, copy paste mistake, wonder why it's not failing the tests; removed.